src/main/java/study/step2/domain/LottoNumbers.java

```diff
@@ -0,0 +1,28 @@
+package study.step2.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbers {
+
+  private static final int LOTTO_MIN_NUMBER = 1;
+  private static final int LOTTO_MAX_NUMBER = 45;
+  private static final int LOTTO_START_RANGE_NUMBER = 0;
+  private static final int LOTTO_END_RANGE_NUMBER = 6;
+
+  private LottoNumbers() {
+  }
+
+  public static List<Integer> pick() {
+    List<Integer> lottoNumbers = IntStream.rangeClosed(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER).boxed().collect(Collectors.toList());
```

Comment:
```suggestion
    List<Integer> lottoNumbers = IntStream.rangeClosed(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
            .boxed()
            .collect(Collectors.toList());
```
Stream 메소드는 한 줄에 한 개만 사용해주세요!